### ISSUE TITLE
prefixtree algo

### ISSUE THREAD

**Issue by missinglink** [pelias/schema#38]
consider changing from using quadtree to geohash

**Comment by sevko** [comment]
Is this still on the cards? Why were we considering switching?

**Comment by missinglink** [comment]
good call, this ticket may become redundant, pending on our discussion regarding storing polygon data inside ES

**Comment by missinglink** [comment]
no longer relevant as we don't store polygon data in ES
